build.sbt

```diff
@@ -20,21 +20,34 @@ updateOptions := updateOptions.value.withCachedResolution(cachedResoluton = true
 val akkaVersion       = "10.1.3"
 val akkaStreamVersion = "2.5.13"
 
+resolvers ++= Seq(
+    Resolver.bintrayRepo("cakesolutions", "maven"),
+    Resolver.jcenterRepo
+)
+
+// Excludes (javax.ws.rs and com.sun.jersey) to fix conflicts for the method: javax.ws.rs.core.MultivaluedMap.addAll
 libraryDependencies ++= Seq(
     "com.typesafe.scala-logging"   %% "scala-logging"          % "3.9.0",
     "ch.qos.logback"               %  "logback-classic"        % "1.2.3"           % Runtime,
     "com.typesafe.akka"            %% "akka-http"              % akkaVersion,
     "com.typesafe.akka"            %% "akka-stream"            % akkaStreamVersion,
     "com.typesafe.akka"            %% "akka-http-spray-json"   % akkaVersion,
     "com.typesafe.akka"            %% "akka-http-testkit"      % akkaVersion,
-    "com.github.swagger-akka-http" %% "swagger-akka-http"      % "0.14.1",
+    "com.github.swagger-akka-http" %% "swagger-akka-http"      % "0.14.1" exclude("javax.ws.rs", "jsr311-api"),
     "com.amazonaws"                %  "aws-java-sdk-s3"        % "1.11.362",
     "com.lightbend.akka"           %% "akka-stream-alpakka-s3" % "0.20",
-    "org.apache.ranger"            % "ranger-plugins-common"   % "1.0.0",
-    "org.scalatest"                %% "scalatest"              % "3.0.5"           % Test,
-    "org.scalamock"                %% "scalamock"              % "4.1.0"           % Test
+    "org.apache.ranger"            % "ranger-plugins-common"   % "1.0.0" exclude("javax.ws.rs", "jsr311-api") exclude("com.sun.jersey", "jersey-core") exclude("com.sun.jersey", "jersey-json") exclude("com.sun.jersey", "jersey-server") exclude("com.sun.jersey", "jersey-bundle"),
```

Comment:
This ranger dependency pulls in Hadoop. And Kafka. And ZooKeeper.

And I got bored reading the full list.

Given we're not using it yet, maybe just remove it entirely?

Reply:
Also, for next time: just `jersey-bundle` should be enough to exclude all the Jersey dependencies.